packages/frontend-2/components/projects/FeedbackRequest/Banner.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div>
+    <div
+      class="flex flex-col space-y-4 sm:space-y-0 sm:space-x-2 sm:items-center sm:flex-row px-4 py-5 sm:py-2 transition hover:bg-primary-muted"
+    >
+      <div class="flex space-x-2 items-center grow text-sm">
+        <div class="text-foreground">
+          How are we doing? Share your feedback in our survey!
+        </div>
+      </div>
+      <div class="flex space-x-2 w-full sm:w-auto shrink-0">
+        <FormButton size="sm" color="default" text @click="showSkipDialog = true">
+          Skip
+        </FormButton>
+        <FormButton
+          size="sm"
+          class="px-4"
+          :icon-right="ArrowTopRightOnSquareIcon"
+          to="https://forms.gle/ngyffiVaZKzEKmtr7"
+          external
+          target="_blank"
+          @click="emitDismissedOrOpened"
+        >
+          Open Survey
+        </FormButton>
+      </div>
+    </div>
+    <ProjectsFeedbackRequestSkipDialog
+      :open="showSkipDialog"
```

Comment:
What if the dialog is closed by the user pressing Esc or clicking outside of it? That is - what if the value of `open` is changed from inside the SkipDialog component and emited out as an event? You've only got one way binding here (no v-model:open and thus not listening for updates to that value), so you're likely to get your local `showSkipDialog` value out of sync from the actual `open` state

Reply:
I added prevent-close-on-click-outside and hide-closer to prevent this. Is this enough, or should I handle additional scenarios where the dialog might be closed unexpectedly?